eessi_bot_event_handler.py

```diff
@@ -138,6 +284,72 @@ def handle_pull_request_event(self, event_info, log_file=None):
         else:
             self.log("No handler for PR action '%s'", action)
 
+    def handle_bot_command(self, event_info, bot_command, log_file=None):
+        """
+        Handle bot command
+
+        Args:
+            event_info (dict): object containing all information of the event
+            bot_command (EESSIBotCommand): command to be handled
+        """
+        cmd = bot_command.command
+        handler_name = f"handle_bot_command_{cmd}"
+        if hasattr(self, handler_name):
+            handler = getattr(self, handler_name)
+            self.log(f"Handling bot command {cmd}")
+            return handler(event_info, bot_command)
+        else:
+            self.log(f"No handler for command '{cmd}'")
+            raise EESSIBotCommandError(f"unknown command '{cmd}'; use `bot: help` for usage information")
+
+    def handle_bot_command_help(self, event_info, bot_command):
+        help_msg = "\n  **How to send commands to bot instances**"
+        help_msg += "\n  - Commands must be sent with a **new** comment (edits of existing comments are ignored)."
+        help_msg += "\n  - A comment may contain multiple commands, one per line."
+        help_msg += "\n  - Every command begins at the start of a line and has the syntax `bot: COMMAND [ARGUMENTS]*`"
+        help_msg += "\n  - Currently supported COMMANDs are: `help`, `build`, `showconfig`"
+        return help_msg
+
+    def handle_bot_command_build(self, event_info, bot_command):
+        gh = github.get_instance()
```

Comment:
Should there be a `self.gh` so we don't need to create a new instance every time?

Not sure that's wise actually, since it'll be long-lived, just a random thought

Reply:
Could be problematic if the token expires, and `github.*` caches the instance already anyway, so nevermind